docs/howtos/cut-a-new-release.md

```diff
@@ -1,39 +1,37 @@
 # Application Services Release Process
 
 ## Make a new release from latest main.
+1. Create a release commit.
+    - **Automated process:** run the `./automation/prepare-release.py` script to create a release commit and open a pull-request.
+    - **Manual process:** if the automated process above fails, refer to the [Create a release commit manually](#create-a-release-commit-manually) section.
 
-1. Smoketest whether the release will integrate cleanly with key downstream consumers:
-    - Run the `./automation/smoke-test-firefox-ios.py` script to test integration with Firefox for iOS.
-    - Run the `./automation/smoke-test-android-components.py` script to test integration with Android Components.
-    - Run the `./automation/smoke-test-fenix.py` script to test integration with Fenix
-    - Build and run the [sync sample app](https://github.com/mozilla-mobile/android-components/tree/master/samples/sync) in android-components
-      against your local checkout using the [local publishing flow](./locally-published-components-in-fenix.md).
-
-    If these tests fail, it indicates a breaking change. Check that such a change is called out
-    in the changelog, and file downstream bugs to track the work of integrating it.
-2. Run the `./automation/prepare-release.py` script to create a release commit and open a pull-request. If this script does not work for you, please refer to the "Create a release commit manually" section in this document.
-3. Cut the actual release.
+    **Note:** Cutting a Github release requires that a tag and build artifacts from your pull request are available so your PR will need to be approved and merged and CI will need to have successfully built your merged changes before a release can be cut.
```

Comment:
I struggled to get my head around this - I suggest simplifying it to something like: "Because the release commit must always be on the `main` branch, your PR will need to be approved, CI successful, and merged before a release can be cut.